src/game/laramisc.c

```diff
@@ -152,6 +153,20 @@ void LaraControl(int16_t item_num)
         Lara.target = NULL;
     }
 
+    int16_t camera_move_delta = PHD_45 / 30;
+
+    if (Input & IN_CAMERA_LEFT) {
```

Comment:
Does this handle correctly people holding "Camera Up" and "Camera Right" at the same time?